www/grid_view/src/module/changeinfo.directive.coffee

```diff
@@ -0,0 +1,8 @@
+class Changeinfo extends Directive
```

Comment:
The default changedetails directive displays too much to my taste. It looks ok when changes are displayed vertically, not horizontally.

Or maybe I can tune the `changedetails` directive to accept an option to hide some info ?